scheduler/config-k8s.edn

```diff
@@ -75,7 +75,20 @@
          :port #config/env-int "COOK_NREPL_PORT"}
  :pools {:default "k8s-gamma"
          :job-resource-adjustment {:adjust-job-resources-fn cook.kubernetes.api/adjust-job-resources
-                                   :pool-regex "^k8s-.+"}}
+                                   :pool-regex "^k8s-.+"}
+         :default-container-for-pool
+         [{:pool-regex "k8s-alpha" :container {:type "docker"
+                                               :docker {
+                                                        :image "python:3.5.9-stretch",
+                                                        :network "HOST"
+                                                        :force-pull-image false
+                                                        :parameters []}}}
+          {:pool-regex "k8s-gamma" :container {:type "docker"
+                                               :docker {
+                                                        :image "python:3.5.9-stretch",
+                                                        :network "HOST"
+                                                        :force-pull-image false
+                                                        :parameters []}}}]}
```

Comment:
Shouldn't we combine these into one entry?

Reply:
Merged